moped-database/migrations/1735340549358_add-funding-status-description/up.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE "public"."moped_fund_status" 
+ADD COLUMN "funding_status_description" text NULL;
+
+COMMENT ON COLUMN "public"."moped_fund_status"."funding_status_description" IS 'Description of the funding status';
+
+UPDATE "public"."moped_fund_status"
+SET "funding_status_description" = CASE "funding_status_name"
+    WHEN 'Tentative' THEN 'In conversation about possible funding commitment'
+    WHEN 'Confirmed' THEN 'Commitment to funding'
+    WHEN 'Available' THEN 'Funding is available, e.g. private developer'
+    WHEN 'Funding setup requested' THEN NULL
+    WHEN 'Set up' THEN 'Funding has been set up in eCAPRIS; has FDU'
+    ELSE "funding_status_description"
+END;
```

Comment:
Looks good in the DB ✅  

It would be cool to fill in the descriptions for the other two statuses, just for symmetry. Maybe:

- Archived: Indicates soft deletion [is that correct? i noticed we hide this from the data dictionary, which is cool]
- Funding setup requested: Requested that funding be set up in eCAPRIS

🤷

Reply:
Yep, I originally thought name of this one was a description in itself, but John's example is making me see that the extra info about where it is setup (in eCapris) could be valuable. @mateoclarke could you update this to John's suggestion? I like it.